src/vector/jitsi/index.ts

```diff
@@ -371,6 +373,7 @@ function joinConference(audioInput?: string | null, videoInput?: string | null):
             apiLogLevels: ["warn", "error"],
         } as any,
         jwt: jwt,
+        lang: language,
```

Comment:
does this Jitsi API support the same language codes as Element?

`en` is an IS 639-1 code but Element uses IETF language tags which is `${ISO 639-1}-${ISO 3166 country code}`

Reply:
Thanks for pointing that out, the documentation of Jitsi isn't that clear about that. Some configuration features are using the latter, but `lang` isn't. Instead they use a format like `enGB` (not sure how that one is called), which is a bit wired because they use i18next which normally expects IETF language tags.

[They have a function that translate](https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/i18n/customNavigatorDetector.ts#L59-L67) from IETF language tags into their format that we could use here too.

A bit funny, on some [configuration options](https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-configuration) they actually use the IETF language tags format, but actually the default value is based on `lang`, so they have a [translation table](https://github.com/jitsi/jitsi-meet/blob/master/react/features/transcribing/jitsi-bcp47-map.json) 😆 

Jitsi seems to fallback correctly to the base language if I pass something like deAU, which Jitsi doesn't have own translations for.

So if we use the translation function we still have one problem: **Jitsi and Element don't agree what `en` means**, one sees it as `en-US`, the other on `en-GB` (based on their headquarter?). We could special case that, e.g. translating `en` to `en-GB` before passing it.  However, I can not guarantee that there aren't other mapping problems.

@t3chguy What is your option on this? Going forward with a transformation function and special casing the `en` case?

Reply:
that sounds sane, can always adjust in the future if it diverges

Reply:
Fixed it, but missed to do a sign-off, can I add it to the PR description, or should I force push the commit?

Reply:
You can add it in the PR description